components/host-dashboard/reports-section/HostDownloadsSection.js

```diff
@@ -50,7 +50,7 @@ const triggerCSVDownload = (host, reportUrl, dateInterval) => {
 };
 
 const HostDownloadsSection = ({ host, collectives, dateInterval }) => {
-  const accountsSlugs = collectives?.map(c => c.value.slug);
+  const accountsSlugs = collectives?.map(c => c.value?.slug);
```

Comment:
I fixed this in https://github.com/opencollective/opencollective-frontend/pull/7110